Dockerfile

```diff
@@ -57,7 +57,8 @@ FROM ${BASE_RUBY_IMAGE_WITH_GEMS_AND_NODE_MODULES} AS assets-precompile
 WORKDIR /app
 COPY . .
 
-RUN yarn jest && \
+RUN yarn run lint && \
+    yarn run test && \
```

Comment:
Should we add a script to `package.json` so this Dockerfile stays as dumb as possible?

Reply:
Oh, what @vigneshmsft said!